static/js/compose.js

```diff
@@ -758,6 +759,19 @@ function validate_private_message() {
         );
         return false;
     }
+
+    for (const user_id of user_ids) {
+        if (!people.is_person_active(user_id)) {
+            context = {email: people.get_by_user_id(user_id).email};
+            compose_error.show(
+                $t_html({defaultMessage: "The user {email} has been deactivated"}, context),
```

Comment:
I changed this string, both for consistency and to add a `.` at the end of the error message.  Can you do a follow-up pass to make that `.` appear at the end of all of these?